### PR TITLE
Change website with new template

### PR DESCRIPTION
**Pull request by thminhVN**
#### Status

READY

#### What's this PR do?

- [x] Change new template in homepage
- [x] Change order from page to popup in menu
- [x] Change new template in tracking page
- [x] Merge page payment and tracking into one page
- [x] Change new template in "Term of Service"  page
- [x] Change new template in "Privacy Policy" page
- [x] Change template order success email
- [x] Change template pay success email
- [x] Show detail table for all time
- [x] Change deposit value from 100% to 50%
- [x] Fix unit test
- [x] Hot deal page

#### What are the relevant Git/YouTrack tickets?

// Put in link to Git Issue or YouTrack ticket

#### Screenshots (if appropriate)
screencapture-localhost-4000-1474644826517
screencapture-localhost-4000-tracking-1474645611241
screencapture-localhost-4000-tracking-1474645764087
screencapture-localhost-4000-khuyen-mai-1474645574196

#### Questions:

- Is there a blog post?
- Does the knowledge base need an update?
- Does this add new dependencies which need to be added to?

### CODE REVIEW

File: web/controllers.ex [hunk 1]
```diff
@@ -83,4 +83,16 @@ defmodule Delivr.Controllers do
     Enum.join(parsed_operators, ", ")
   end
 
+  @doc """
+
+  """
+  def format_quantity(quantity) do
+    intQuantity = round quantity
+    if quantity - intQuantity !=  0 do
```
Comment (thminhVN) [inline web/controllers.ex]: add empty line before this line

File: web/controllers/order_controller.ex [hunk 2]
```diff
@@ -437,7 +394,7 @@ defmodule Delivr.OrderController do
       {:ok, url} ->
         url
 
-      {:error, _changeset} ->
+      {:error, changeset} ->
```
Comment (thminhVN) [inline web/controllers/order_controller.ex]: add underscore

File: web/controllers/order_controller.ex [hunk 3]
```diff
@@ -513,7 +469,7 @@ defmodule Delivr.OrderController do
         Email.build()
         |> Email.put_from(Application.get_env(:delivr, :cs_email), Application.get_env(:delivr, :branch_name))
         |> Email.add_to(order.email, order.name)
-        |> Email.put_subject("Đơn hàng #{order.order_code} của bạn đã đặt cọc thành công")
+        |> Email.put_subject("Đơn hàng #{order.order_code} của bạn đã thanh toán thành công")
```
Comment (thminhVN) [inline web/controllers/order_controller.ex]: upcase order.order_code

**Comment by hieuphq** [comment]
👍

**Pull request merged**